CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- File pointer errors on issues on the first line of files where BOM is present.
```

Comment:
This reads a little stilted - maybe something like this?

```suggestion
- File pointer errors when an issue is raised on the first line of a file with a byte order mark.
```

Reply:
Sure, I like the suggested version so I'll just use that directly.